backend/app/transcendence/nest-cli.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://json.schemastore.org/nest-cli",
+  "collection": "@nestjs/schematics",
+  "sourceRoot": "src",
+  "compilerOptions": {

```

Comment:
{ on end of line.

Reply:
yep, automatically generated file, will be fixed